src/test/resources/gff3toff_rules/multiple_entries_predefined_regions.gff3

```diff
@@ -0,0 +1,11 @@
+##gff-version 3.1.26
+##sequence-region BN000065.1 1 315242
+##sequence-region BN000066.1 1 315242
+
+BN000065.1	.	gene	1	315242	.	+	.	ID=gene_RHD;gene=RHD;
+BN000065.1	.	mRNA	133806	191728	.	+	.	Parent=gene_RHD;
+BN000065.1	.	rRNA	133970	145841	.	+	.	Parent=gene_RHD;number=1;
```

Comment:
Add some tests with joins, complements, etc

Reply:
The logic we changed here mainly affects the construction of the Gff3Annotation, in particular defining the boundary of when one annotation starts and the previous one ends. The join, complements etc are handled by the GFF3Mapper, which works at a higher level by making use of the annotation. 

This means that adding extra checks will be redundant and may actually muddle the testing files.